src/nuclide_header.F90

```diff
@@ -265,11 +267,31 @@ subroutine nuclide_from_hdf5(this, group_id, temperature, method, tolerance)
                &for " // trim(this % name) // " at or near " // &
                trim(to_str(nint(temp_desired))) // " K.")
         end if
-      end do TEMP_LOOP
+      end do
 
     case (TEMPERATURE_INTERPOLATION)
-      ! TODO: Get bounding temperatures
-      call fatal_error("Temperature interpolation not yet implemented")
+      ! If temperature interpolation or multipole is selected, get a list of
+      ! bounding temperatures for each actual temperature present in the model
+      TEMP_LOOP: do i = 1, temperature % size()
+        temp_desired = temperature % data(i)
+
+        do j = 1, size(temps_available) - 1
```

Comment:
I see two 'easy' options:
1) Just switch from `interpolation` to `nearest` (with a warning) if this routine detects there is only one temperature in one of the nuclide's datasets?  That's naiive but safe.
2) For every nuclide that has only one temperature (that is the right temperature), make a fake upper temperature (that has as little data as possible) so that interpolation routines work, but always pick the right temperature.